openid-sharedsignals-framework-1_0.md

```diff
@@ -1981,8 +1981,7 @@ This section is a profile of the following IETF SecEvent specifications:
 The RISC use cases that set the requirements are described in Security Events
 RISC Use Cases {{USECASES}}.
 
-The CAEP use cases that set the requirements are described in CAEP Use Cases (TODO: Add
-        reference when file is added to repository.)
+The CAEP use cases that set the requirements are described in CAEP Use Cases {{CAEP}}.
```

Comment:
This was supposed to be a separate use-cases doc. There was a working doc, but it never made it to the repo (needed more work I guess)

Reply:
Probably you are referring to [this](https://docs.google.com/document/d/1tmMqiXNB-lW9HXIzrivOvaFSts23zAzKLWPcSD740kE/edit#heading=h.fsduc31pruxn)